src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildWithLibraryTests.cs

```diff
@@ -269,11 +269,7 @@ public void ProjectDependencies ([Values(true, false)] bool projectReference)
 			};
 			libB.Sources.Clear ();
 			libB.Sources.Add (new BuildItem.Source ("Foo.cs") {
-				TextContent = () => @"public class Foo {
```

Comment:
I don't immediately understand this change, and I don't like it. I think having `Bar` created *somewhere* would be beneficial, though maybe it needs to be "elsewhere"?

Reply:
This changes the inheritance to be like https://github.com/dotnet/maui/issues/18819

* Assembly with `Foo : Bar`, has no `Mono.Android.dll` reference.
* `Bar` is a `Java.Lang.Object`

Previously it just created a `Bar` in the constructor, but that won't reproduce our problem.